src/makeCalculator.js

```diff
@@ -38,6 +38,40 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+
+    result: 0,
+
+    operate(operation, operand) {
+      operation(operand);
+
+      return this;
+    },
+
+    add(value) {
+      calculator.result += value;
```

Comment:
```suggestion
      this.result += value;
```

Reply:
and further through the code

Reply:
Agree with @roma-shafihullin here )

When you use constructors, classes, or something like that `this` looks to service which call your method
`calculator.reset()` -> this on reset is a calculator
But add is a callback, the function doesn’t have this:
`add()` -> this isn’t exit here

```suggestion
    reset() {
      this.result = 0;

      return this;
    },
```